k8s/values.yaml

```diff
@@ -1,4 +1,4 @@
 name: dino-park-dev-tools
-namespace: dino-park
+namespace: dinopark-dev
```

Comment:
Hm, this widens the gap between dev and prod. Though, I guess this is a double check for making sure we're in the right environment?

Are these expected to be ran in the same K8s cluster?